bin/otomi

```diff
@@ -93,7 +92,7 @@ function show_usage() {
   Env flags:
     CI=1                Exit the script after first error
     DEBUG=1             Run otomi CLI in debug mode
-    TIME=1              Time otomi CLI calls
+    TRACE=1              Time otomi CLI calls
```

Comment:
Copy paste issue. Please provide correct description.

Reply:
done

Reply:
Thank you!